admin/javascript/src/components/breadcrumb/index.js

```diff
@@ -24,7 +24,7 @@ class BreadcrumbComponent extends SilverStripeComponent {
      ]),
        this.props.crumbs.slice(-1).map((crumb, index) => [
          <li className="breadcrumb__item breadcrumb__item--last">
-           <h2 className="breadcrumb__item-title breadcrumb__item-title--last" key={index}>
+           <h2 className="breadcrumb__item-title breadcrumb__item-title--last text-truncate" key={index}>
```

Comment:
Adding a global `text-truncate` modifier is a bit of a BEM antipattern: http://getbem.com/faq/#can-i-create-global-modifier. Can we use a SCSS include instead, and apply it to `breadcrumb__item--title`?